server/src/common/extensions.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
See note above about header comments. Please stay consistent with the existing pyright header comment contents and format. I feel very strongly that every code file should have a short description of its contents — the reason for the file's existence. It's really important for understanding a code base and helps encourage good thinking about modularity.